schemas/database/001_initial.sql

```diff
@@ -8,3 +8,23 @@ CREATE TABLE transactions (
     raw         BYTEA                    NOT NULL,
     hash        BYTEA                        NULL
 );
+
+CREATE TABLE log_fetches
```

Comment:
I somehow don't like how this tables turns random whims of the API into a permanent append-only record. One way to make it more palatable is if it could merge adjacent ranges. So when everything is synced up and done this table would just contain one `[0..latest_block]` range spanning the entire chain.

But if all goes well this table will remain small, so no need to add substantial complexity for this.